packs/data/spells.db/vicious-jealousy.json

```diff
@@ -0,0 +1,99 @@
+{
+    "_id": "h6VoHgPC0JCTVzgP",
+    "data": {
+        "ability": {
+            "value": ""
+        },
+        "area": {
+            "areaType": "",
+            "value": ""
+        },
+        "category": {
+            "value": "spell"
+        },
+        "components": {
+            "focus": false,
+            "material": false,
+            "somatic": true,
+            "verbal": true
+        },
+        "cost": {
+            "value": ""
+        },
+        "damage": {
+            "value": {}
+        },
+        "description": {
+            "value": "<p>The target is overcome by deep jealousy and resentment that twists its mind against other creatures. It must attempt a Will save.</p>\n<hr />\n<p><strong>Critical Success</strong> The target is unaffected.</p>\n<p><strong>Success</strong> For 1 round, the target no longer treats anyone as its allies. The spell then ends.</p>\n<p><strong>Failure</strong> The target no longer treats anyone as its allies. The target can’t take the @Compendium[pf2e.actionspf2e.Aid]{Aid} reaction or any action that would directly benefit another creature, such as casting a beneficial spell on them, though it can still take actions that indirectly benefit other creatures, such as fighting the same foes.</p>\n<p><strong>Critical Failure</strong> As failure, and whenever the target starts its turn within reach of a creature it previously considered an ally, it must spend its first action on its turn to berate the creature or otherwise wave it away.</p>\n<hr />\n<p><strong>Heightened (4th)</strong> You can target up to 10 creatures.</p>"
```

Comment:
```suggestion
            "value": "<p>The target is overcome by deep jealousy and resentment that twists its mind against other creatures. It must attempt a Will save.</p>\n<hr />\n<p><strong>Critical Success</strong> The target is unaffected.</p>\n<p><strong>Success</strong> For 1 round, the target no longer treats anyone as its allies. The spell then ends.</p>\n<p><strong>Failure</strong> The target no longer treats anyone as its allies. The target can't take the @Compendium[pf2e.actionspf2e.Aid]{Aid} reaction or any action that would directly benefit another creature, such as casting a beneficial spell on them, though it can still take actions that indirectly benefit other creatures, such as fighting the same foes.</p>\n<p><strong>Critical Failure</strong> As failure, and whenever the target starts its turn within reach of a creature it previously considered an ally, it must spend its first action on its turn to berate the creature or otherwise wave it away.</p>\n<hr />\n<p><strong>Heightened (4th)</strong> You can target up to 10 creatures.</p>"
```